ext/pcre/php_pcre.c

```diff
@@ -645,28 +645,30 @@ PHPAPI pcre_cache_entry* pcre_get_compiled_regex_cache_ex(zend_string *regex, in
 	}
 
 	p = ZSTR_VAL(regex);
+	end_p = ZSTR_VAL(regex) + ZSTR_LEN(regex);
```

Comment:
Since we're no longer restricted to C89, maybe declare the variable here:
```suggestion
	const char *end_p = ZSTR_VAL(regex) + ZSTR_LEN(regex);
```

Reply:
Applied this in the latest commit as well.